Manylinux2014_Compliant_Source/pkg/build.sh

```diff
@@ -18,8 +18,8 @@ cd ..
 cmake -DCMAKE_BUILD_TYPE=Release ../src && make -j"$(nproc)" && make install
 
 cd ..
-cp src/include/utils.h Manylinux2014_Compliant_Source/pkg/IR2Vec/
+cp src/include/*.h Manylinux2014_Compliant_Source/pkg/IR2Vec/
 cp build/version.h Manylinux2014_Compliant_Source/pkg/IR2Vec/
-cp vocabulary/seedEmbeddingVocab-*.txt Manylinux2014_Compliant_Source/pkg/IR2Vec/
+cp vocabulary/seedEmbeddingVocab-llvm14.txt Manylinux2014_Compliant_Source/pkg/IR2Vec/
```

Comment:
IMO, this way of naming is better instead of inserting an asterisk. That method is prone to errors. 

In `testIR2Vec.py`, we have 
```
SEED_VERSION = (
    list(IR2Vec.preparation.install_loc.glob("seed*.txt"))[0]
    .name.split(".")[0]
    .split("-")[-1]
)
```

If in future, for any reason, we have multiple files with this naming scheme, the code becomes error prone.

Reply:
However, if it's about the specific naming of llvm14, we can change it to a different neutral name

Reply:
Is it better to have it just as `seedEmbeddingVocab.txt`?

Reply:
Yea, that can be done